packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -73,27 +74,28 @@ class _TextSelectionToolbar extends StatelessWidget {
   Widget build(BuildContext context) {
     final List<Widget> items = <Widget>[];
     final Widget onePhysicalPixelVerticalDivider =
-        SizedBox(width: 1.0 / MediaQuery.of(context).devicePixelRatio);
+    SizedBox(width: 1.0 / MediaQuery.of(context).devicePixelRatio);
+    final DefaultCupertinoLocalizations localizations = CupertinoLocalizations.of(context);
```

Comment:
This type should be `CupertinoLocalizations` i think right? That way we can have different subclasses provided by a localized bundle